tests/ZendTest/Db/Adapter/Platform/PostgresqlTest.php

```diff
@@ -74,13 +74,24 @@ public function testGetQuoteValueSymbol()
     /**
      * @covers Zend\Db\Adapter\Platform\Postgresql::quoteValue
      */
-    public function testQuoteValue()
+    public function testQuoteValueRaisesNoticeWithoutPlatformSupport()
     {
         $this->setExpectedException(
-            'PHPUnit_Framework_Error',
+            'PHPUnit_Framework_Error_Notice',
             'Attempting to quote a value in Zend\Db\Adapter\Platform\Postgresql without extension/driver support can introduce security vulnerabilities in a production environment'
         );
-        $this->assertEquals("'value'", $this->platform->quoteValue('value'));
+        $this->platform->quoteValue('value');
+    }
+
+    /**
+     * @covers Zend\Db\Adapter\Platform\Postgresql::quoteValue
+     */
+    public function testQuoteValue()
+    {
+        $this->assertEquals("E'value'", @$this->platform->quoteValue('value'));
+        $this->assertEquals("E'Foo O\\'Bar'", @$this->platform->quoteValue("Foo O'Bar"));
+        $this->assertEquals('E\'\\\'; DELETE FROM some_table; -- \'', @$this->platform->quoteValue('\'; DELETE FROM some_table; -- '));
+        $this->assertEquals("E'\\\\\\'; DELETE FROM some_table; -- '", @$this->platform->quoteValue('\\\'; DELETE FROM some_table; -- '));
```

Comment:
Can you please check this against the changes introduced to fix [ZF2015-02](http://framework.zend.com/security/advisory/ZF2015-02)? I'm pretty sure we changed escaping to better match the recommendations made by PostgreSQL.